x-pack/plugins/uptime/public/components/queries/error_list/error_list.tsx

```diff
@@ -22,38 +24,74 @@ export const ErrorList = ({ dateRangeStart, dateRangeEnd, filters }: ErrorListPr
   <Query query={getErrorListQuery} variables={{ dateRangeStart, dateRangeEnd, filters }}>
     {({ loading, error, data }) => {
       if (loading) {
-        return 'Loading...';
+        return i18n.translate('xpack.uptime.errorList.loadingMessage', {
+          defaultMessage: 'Loading...',
+        });
       }
       if (error) {
-        return `Error ${error.message}`;
+        return i18n.translate('xpack.uptime.errorList.errorMessage', {
+          values: { message: error.message },
+          defaultMessage: 'Error {message}',
+        });
       }
       const { errorList } = data;
       return (
         <Fragment>
           <EuiTitle size="xs">
-            <h5>Error list</h5>
+            <h5>
+              <FormattedMessage id="xpack.uptime.errorList.title" defaultMessage="Error list" />
+            </h5>
           </EuiTitle>
           <EuiPanel>
             <EuiInMemoryTable
               items={errorList}
               columns={[
-                { field: 'type', name: 'Error Type', sortable: true },
+                {
+                  field: 'type',
+                  name: i18n.translate('xpack.uptime.errorList.columns.type', {
+                    defaultMessage: 'Error type',
+                  }),
+                  sortable: true,
+                },
                 {
                   field: 'monitorId',
-                  name: 'Monitor ID',
+                  name: i18n.translate('xpack.uptime.errorList.columns.monitorId', {
+                    defaultMessage: 'Monitor ID',
+                  }),
                   render: (id: string) => <Link to={`/monitor/${id}`}>{id}</Link>,
                   sortable: true,
                   width: '25%',
                 },
-                { field: 'count', name: 'Count', sortable: true },
+                {
+                  field: 'count',
+                  name: i18n.translate('xpack.uptime.errorList.columns.count', {
+                    defaultMessage: 'Count',
+                  }),
+                  sortable: true,
+                },
                 {
                   field: 'timestamp',
-                  name: 'Latest error',
+                  name: i18n.translate('xpack.uptime.errorList.columns.latestError', {
+                    defaultMessage: 'Latest error',
+                  }),
                   sortable: true,
                   render: (timestamp: string) => moment(timestamp).fromNow(),
                 },
-                { field: 'statusCode', name: 'Response code', sortable: true },
-                { field: 'latestMessage', name: 'Latest message', sortable: true, width: '40%' },
+                {
+                  field: 'statusCode',
+                  name: i18n.translate('xpack.uptime.errorList.columns.responseCode', {
```

Comment:
Nit, doesn't need to be resolved in this PR, but any reason we prefer "Response Code" to status code? If we can stick with one that'll be less confusing. The HTTP spec calls them status.

Reply:
++ - it should be consistent with the actual field name, especially since this is the key and not visible copy. Will update.

Reply:
Addressed in https://github.com/elastic/kibana/pull/28183/commits/994eb8cf052e8b0e41010e58135eef94d92163e7